src/components/InvertedFlatList/index.native.tsx

```diff
@@ -1,9 +1,10 @@
-import React, {forwardRef} from 'react';
+import React, {ForwardedRef, forwardRef} from 'react';
```

Comment:
I guess, we shouldn't be importing React here if it is not being used anywhere in the code.